scout/commands/update/hpo.py

```diff
@@ -43,8 +43,9 @@ def hpo(hpoterms, hpo_to_genes):
     adapter = store
 
     LOG.info("Dropping HPO terms")
-    adapter.hpo_term_collection.drop()
+    adapter.hpo_term_collection.delete_many({})
```

Comment:
You have incorporated also this change from my previous PR. But it should be fine when we merge #3016

Reply:
Yes, that was intentional! All good!